src/service/arguments/cli/tools/resume.ts

```diff
@@ -0,0 +1,24 @@
+import { Command } from "commander";
+import { CommandConstructor } from "@bc/service/arguments/cli/command-constructor";
+import { ToolType } from "@bc/domain/cli";
+import Container from "typedi";
+import { InputService } from "@bc/service/inputs/input-service";
+
+/**
+ * Create command parser for project list tool
+ * @implements {CommandConstructor}
+ */
+export class ResumeCommand implements CommandConstructor {
+  createCommand(): Command {
+    const program = new Command(ToolType.RESUME);
+    program
+      .description("Resume execution from first point of failure")
+      .option("-w, --workspace <workspace>", "The workspace in which build-chain was executed and the one you want to resume execution in")
```

Comment:
avoiding `you` `I` usage
```suggestion
      .option("-w, --workspace <workspace>", "The workspace in which build-chain was executed and the one to resume execution in")
```